contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -48,11 +48,35 @@ class DownloadGitRepoAction(Action):
     def __init__(self, config=None, action_service=None):
         super(DownloadGitRepoAction, self).__init__(config=config, action_service=action_service)
 
+        self.https_proxy = os.environ.get('https_proxy', self.config.get('https_proxy', None))
+        self.http_proxy = os.environ.get('http_proxy', self.config.get('http_proxy', None))
```

Comment:
pip doesn't have a --no-proxy CLI option but requests grabs the env variable directly  https://github.com/pypa/pip/issues/2440. So I made a change to just set the env variable. It's completely messed up.